docs/testnet/start-testnet-tutorial.md

```diff
@@ -0,0 +1,385 @@
+## Interchain-Security Testnet
+
+This guide contains the instructions to setup a Interchain-Security Testnet. For the sake of simplicity, both provider and consumer chains run a single node chorum. After the completition of this tutorial you have the possibility to add other nodes to the networks by following these complementary intructions [guide](https://github.com/sainoe/IS-testnet/blob/main/join-is-tesnet.md).
+
+---
+
+### Prerequisites
+- Go 1.18+ <sub><sup>([installation notes](https://go.dev/doc/install))<sub><sup>
+- Interchain-Security binaries
+- Rust 1.60+ <sub><sup>([installation notes](https://www.rust-lang.org/tools/install))<sub><sup>
+- Hermes v1.0.0-rc.0
+- jq
+
+---
+
+### Install the IS Binary
+```
+git clone https://github.com/cosmos/interchain-security.git
+cd interchain-security
+make install
+```
+
+---
+
+### Install the IBC-Relayer
+Follow the instruction to install the IBC-Relayer Rust implementation [here](https://hermes.informal.systems/getting_started.html).
+
+---
+
+### Provider chain setup
+The following steps explain how to setup a provider chain running a single validator node. Choose a directory name `/<prov-node-dir>` (e.g. `~/provider`) to store the provider chain node files. Along this guide, the provider and consumer chain ids used are respectively `provider` and `consumer`.
+
+__0. Remove any existing node directory__
+
+`rm -rf /<prov-node-dir>`<br/><br/>
+
+__1. Create an initial genesis__  
+Create the initial genesis file (`/<prov-node-dir>/config/genesis.json`) with the following command:  
+
+`interchain-security-pd init <provider-node-moniker> --chain-id provider --home /<prov-node-dir>`  
+
+*If you want to make any updates to the genesis, it is a good opportunity to make these updates now.*<br/><br/>
+
+__2. Reduce proposal voting period__  
+This command will shorten the voting period to 3 minutes in order to make pass a consumer chain proposal.
+
+```
+jq ".app_state.gov.voting_params.voting_period = \"180s\"" \
+    /<prov-node-dir>/config/genesis.json > /<prov-node-dir>/edited_genesis.json
+
+mv /<prov-node-dir>/edited_genesis.json /<prov-node-dir>/config/genesis.json
+```  
+<br/><br/>
+
+__3. Create an account keypair__  
+This following step creates a public/private keypair and stores it under the given keyname. The output is also exported into a json file for later use.  
+```
+interchain-security-pd keys add <provider-keyname> --home /<prov-node-dir> \
```

Comment:
Replace `<provider-keyname>` with actual keyname and mention in the description that the command is for **_keyname_** `provider-key` or something. In general, make the commands so that they can be copy pasted